homeassistant/components/zha/strings.json

```diff
@@ -226,7 +226,7 @@
       "dim_down": "Dim down",
       "left": "Left",
       "right": "Right",
-      "open": "Open",
+      "open": "[%key:common::state::open%]",
```

Comment:
This would be an action and not a state so don't think it's right to change this here.

Reply:
Right, may be we should create common actions instead.